frappe/core/doctype/doctype/doctype.json

```diff
@@ -660,8 +661,15 @@
    "fieldtype": "Tab Break",
    "label": "Connections",
    "show_dashboard": 1
+  },
+  {
+   "fieldname": "filters",
+   "fieldtype": "Table",
+   "label": "filters",
```

Comment:
"Link Filters" maybe, for a little more context?
Or even "Inbound Link Filters"?